packages/cli/test/integration.js

```diff
@@ -3277,6 +3277,40 @@ test('deploy gatsby twice and print cached directories', async t => {
   }
 });
 
+test('deploy pnpm twice using pnp and symlink=false', async t => {
+  const directory = path.join(__dirname, 'fixtures/unit/pnpm-pnp-symlink');
+
+  await fs.rm(path.join(directory, '.vercel'), {
```

Comment:
This is failing in CI. I think you either need the real `fs` instead of `fs-extra`. Or keep `fs-extra` and use `remove()`